### PR TITLE
Split Manual contents into additional pages

### LINKED ISSUE #1267: Update html to Reorganize Manual (redesign Phase 1)
Apply the design changes from @michellelsfong 's design ( .sketch file is linked in the slack channel).

### PR DESCRIPTION
**Pull request by wittejm**
New pages: Privacy Policy (with license and credits), Appendix, and FAQ

Link the new pages in the Footer; adjusted text size and spacing in the footer.

Updated spacing in the Manual nav panel

Using hidden h1 tags for page titles (verify with @hmarcks on best practice on usage/placement of h1)

Moved the Header component outside of individual pages into `<App>`.

Replaced `<Link>` (react) components in the Header with `<a>` because Link doesn't work outside the react router.

resolves #1267

### CODE REVIEW

**Comment by wittejm** [comment]
@hmarcks I have two questions, because you've said before that having  `<h1>`: on every page is a good idea:

The updated mockup removes the small "Manual" title from the Manual page. Should this `<h1>` remain where it is but be `visually-hidden`? Or, should I change the first section header `Introduction` to use an `<h1>`?

Second, the section headers for these pages all appear in inside the `<main><article>` nested containers, but the `<h1>` header is above. Should the `h1` tag stay above this main body container for HTML best-practice reasons? If so I can shift things around to pull the section header up without changing the visuals much.

**Comment by hmarcks** [comment]
If it's intentional that the "Manual" title is removed, then yeah I think a visually-hidden <h1> is best practice. The reason I question if it is intentional is because the other pages have titles - @michellelsfong can you please confirm?

I think the h1 is fine outside the container. There should only be a [single h1 per page](https://www.24a11y.com/2018/the-d-in-the-dom/) though even though the spec allows it.

**Comment by wittejm** [comment]
Gotcha. And just to be explicit, is it ok if the `<h1>` for Faq/Privacy/Appendix  reside *inside* thr `<main><article>` instead?

**Comment by michellelsfong** [comment]
Confirming that the small header was intentionally removed 👍

**Comment by hmarcks** [comment]
Yeah you can nest it in the `<main><article>`

**Comment by wittejm** [comment]
Ok I think I got all the spacing right. @michellelsfong do you have the project installed locally? I think that's the best way to review my changes if you just checkout this branch.

**Comment by KentShikama** [comment]
Looks good to me. Here are some screenshots.

Manual

Screenshot_2020-06-25 RecordSponge Oregon

Privacy Policy

Screenshot_2020-06-25 RecordSponge Oregon(1)

FAQ

Screenshot_2020-06-25 RecordSponge Oregon(2)

Appendix

Screenshot_2020-06-25 RecordSponge Oregon(3)

File: src/frontend/src/components/Footer/index.tsx [hunk 1]
```diff
@@ -4,9 +4,15 @@ export default class Footer extends React.Component {
...
-        <a className="link underline hover-blue" href="/manual#privacypolicy">
+        <a className="link hover-blue underline pr5" href="/privacy-policy">
           Privacy Policy
         </a>
+        <a className="link hover-blue underline pr5 pl4" href="/faq">
+          FAQ
+        </a>
+        <a className="link hover-blue underline pl4" href="/appendix">
+          Appendix
+        </a>
```
Comment (hmarcks) [inline src/frontend/src/components/Footer/index.tsx:15]: The footer gets a bit tangled on skinnier screens:
Screen Shot 2020-06-25 at 8 09 52 PM

```suggestion
        <a className="dib link hover-blue bb mr5 mb4" href="/privacy-policy">
          Privacy Policy
        </a>
        <a className="dib link hover-blue bb mr5 mb4" href="/faq">
          FAQ
        </a>
        <a className="dib link hover-blue bb" href="/appendix">
          Appendix
        </a>
```

**Comment by michellelsfong** [comment]
Thanks for the screenshots! Assuming that nav bar is at the top, manual spacing looks good.

A few things:
- Can we remove that divider "Use RecordSponge"? It doesn't need to be there since there's already a sense of separation.
- I see the appendix and stock form links under File for **Expungement > Selecting the correct form** are dotted, is it updated to take users to the new page? There's another under Use RecordSponge > Overview > #6 and potentially #7.
- Seeing the footer now, can we reduce the spacing between links by half? The footer is taller than when I first saw it

**Comment by wittejm** [comment]
@hmarcks I updated all the requested tweaks.

And I fiddled around with the autoformatter to disable it on the Landing page, since that would have caused hella merge conflicts with the open Feature branch. I did still need to touch the Landing page a little to simplify the Header.

**Comment by hmarcks** [comment]
Looks good to me! Confirmed the issues that @michellelsfong highlighted are resolved.

Regarding the landing page autoformatting, I appreciate it, but I'm thinking you can go ahead and enable it for the landing page if you want since I'm just blowing away the current page anyway so conflicts shouldn't be an issue.

**Pull request merged**